adapter/inventory.bzl

```diff
@@ -14,15 +14,16 @@ def _inventory_gen(name, packages, out):
 
 DEPS = [
     "//pkg/adapter:go_default_library",
-
+    "//adapter/kubernetes:go_default_library",
+    "//adapter/noopLegacy:go_default_library",
 ]
 
 def inventory_library(name, packages, deps):
   _inventory_gen("inventory_file_gen", packages, "inventory.gen.go")
 
   go_library(
       name = name,
-      srcs = ["inventory.gen.go"],      
+      srcs = ["inventory.gen.go", "inventory.go"],
```

Comment:
@kyessenov we prefer lists to span multiple lines. Did buildifier not catch this ?

Reply:
this line should be manually written, gazelle / buildifier does not correct a pattern like this (maybe because it's under a `def`)?

Reply:
Yeah, it's a macro so buildifier does not verify the expansion.

Reply:
Isn't this a problem anyway?  Do we want both "inventory.go" and "inventory.gen.go" in this library now?  This seems like a bigger problem than the lists spanning multiple lines.

Reply:
It is not a problem at all. Actually they are already in a single `//adapter:go_default_library` by using `library=` param in `adapter/BUILD` (https://github.com/istio/mixer/blob/master/adapter/BUILD#L11), this does not change anything around the library structure.

Also it's worth noting that the `library=` param in `go_library` is an undocumented hack and will not work anymore with the latest `rules_go`.

Reply:
But, having `inventory.go` in the `bzl` file seems wrong. That file is specifically about generation. Putting non-generated code into that script seems incorrect.

It is fine if we want to clean that up when `inventory.go` is completely deleted, but in the meantime, it just seems to be inverted.

Reply:
`library` is deprecated. I'm simply trying to use bzl/go conventions here, I guess someone else can fix it later who knows the purpose of this macro.

Reply:
wondering who still needs `inventory.go`. Can we remove it now?

Reply:
We still use the mixer1 Kubernetes adapter which is included in inventory.